src/pages/Transactions/transactionUtils.tsx

```diff
@@ -50,6 +50,19 @@ const isUserRejectionError = (error: any) => {
   return false;
 };
 
+const isLedgerError = (error: any) => {
+  if (error.code === -32603) {
+    // Error message when rejecting from ledger or blind signing and debug not enabled
+    if (
```

Comment:
Obviously could do `if (... && ...)` but followed commenting pattern of `isUserRejectionError`

Reply:
Hey @valefar-on-discord do you happen to know the error code that results from not having the proper Ledger settings enabled? Adding error handling here seems like a minimally invasive and safe improvement.